lib/membrane_audio_mixer_bin.ex

```diff
@@ -20,55 +20,28 @@ defmodule Membrane.AudioMixerBin do
 
   require Membrane.Logger
 
+  alias __MODULE__.MixerOptions
   alias Membrane.{Pad, ParentSpec, AudioMixer}
   alias Membrane.Caps.Audio.Raw
   alias Membrane.Caps.Matcher
 
   @supported_caps {Raw,
                    format: Matcher.one_of([:s8, :s16le, :s16be, :s24le, :s24be, :s32le, :s32be])}
 
-  def_options max_node_degree: [
+  def_options max_inputs_per_node: [
                 type: :int,
                 description: """
-                Maximum number of inputs to a single mixer in the mixres tree. Must be at least 2.
+                The maximum number of inputs to a single mixer in the mixers tree. Must be at least 2.
                 """,
                 default: 10
               ],
-              inputs: [
-                type: :int,
-                description: """
-                Number of all input files to be mixed.
-                """
-              ],
-              caps: [
-                type: :struct,
-                spec: Raw.t(),
+              mixer_options: [
+                type: :any,
```

Comment:
I guess this should be somehow documented, cuz I was explaining that recently to Daniel, but `type` here is a bit of leftover from plans to create a CLI tool to start pipelines and allow parsing options - the type was meant to define how to parse the option.
Currently, however, all it does is providing a default spec for types mentioned [here](https://hexdocs.pm/membrane_core/Membrane.Element.Base.html#def_options/1) and a default `inspector` function for `Membrane.Time` (to print default value in docs as e.g. `2 |> Membrane.Time.secons())` instead of `2000000000`).
So:
1. If defining a spec manually, type can be omitted
2. Putting `type` as `:any` doesn't make sense, but there is a more appropriate option - `:struct`